### ISSUE TITLE
`unnecassary_stack`, `unnecassary_row`, and `unnecassary_column`

### ISSUE THREAD

**Issue by Number-3434** [dart-lang/linter#4843]
Lint that informs when a [`Stack`](https://api.flutter.dev/flutter/widgets/Stack-class.html), [`Row`](https://api.flutter.dev/flutter/widgets/Row-class.html), or [`Column`](https://api.flutter.dev/flutter/widgets/Column-class.html) only has a single child in the `children` parameter.

```dart
// BAD, has only one child in children
const Stack(children: [Text('Hello World!')]),
const Row(children: [Text('Hello World!')]),
const Column(children: [Text('Hello World!')]),

// GOOD, has more than one child in children
const Stack(children: [Text('Hello World!'), Text('Hello World Again!')]),
const Row(children: [Text('Hello World!'), Text('Hello World Again!')]),
const Column(children: [Text('Hello World!'), Text('Hello World Again!')]),
```

**Comment by pq** [comment]
@goderbauer

**Comment by goderbauer** [comment]
This would be sort of similar to the `avoid_unnecessary_containers`. I wouldn't be opposed to this lint existing, but I also wonder how much value it would bring. Would be interesting to see how many unnecessary widgets of those types exist in a larger code base. Unlike unnecessary containers I haven't seen many (any?) of these cases in code reviews.
